packages/terra-slider/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "terra-slider",
+  "main": "lib/Slider.js",
+  "version": "1.0.0",
+  "description": "Terra Slider component provides users a way to select a value from a given range by sliding a thumb along a bar to track the value.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-framework.git",
+    "directory": "packages/terra-slider"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "Framework",
+    "terra-slider",
+    "slider",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-framework/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-framework#readme",
+  "peerDependencies": {
+    "react": "^16.8.5",
+    "react-dom": "^16.8.5",
+    "react-intl": "^2.8.0 || 3 || 4 || 5"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "keycode-js": "^3.1.0",
```

Comment:
It does not look like this dependency is being used.

Reply:
removed